react/src/pages/StartPage.tsx

```diff
@@ -0,0 +1,227 @@
+import ActionItemContent from '../components/ActionItemContent';
+import BAIBoard from '../components/BAIBoard';
+import { useWebUINavigate } from '../hooks';
+import { useBAISettingUserState } from '../hooks/useBAISetting';
+import {
+  AppstoreAddOutlined,
+  FolderAddOutlined,
+  LinkOutlined,
+  PlaySquareOutlined,
+} from '@ant-design/icons';
+import { BoardProps } from '@cloudscape-design/board-components/board';
+import { theme } from 'antd';
+import _ from 'lodash';
+import { useState } from 'react';
+
+// import { useTranslation } from 'react-i18next';
+
+export interface StartItem
+  extends BoardProps.Item<{
+    content: JSX.Element;
+  }> {
+  id:
+    | 'createFolder'
+    | 'startSession'
+    | 'startBatchSession'
+    | 'modelService'
+    | 'startFromExample'
+    | 'startFromURL';
+}
+
+const StartPage: React.FC = () => {
+  // const { t } = useTranslation();
+  const { token } = theme.useToken();
+  const webuiNavigate = useWebUINavigate();
+
+  const defaultSummaryElements: {
+    [key in StartItem['id']]: StartItem['data'];
+  } = {
+    createFolder: {
+      // content: <SummaryItemStartMenu allowNeoSessionLauncher />,
+      content: (
+        <ActionItemContent
+          title="Create a New Storage Folder"
+          description="Creating a folder and uploading files are crucial for training models or providing services externally. Please begin by uploading the files."
+          buttonText="Create Folder"
```

Comment:
Let's use i18n

Reply:
Added translation. The wording is not finalized yet, so we need @YEONFEEL96  to confirm.

Can you please check the wording of en.json and en.json under resource/i18n? @YEONFEEL96